app.js

```diff
@@ -40,58 +62,78 @@ storeSalesDataEl.appendChild(timeRowEl);
 var blankCell = document.createElement('td');
 timeRowEl.appendChild(blankCell);
 
-for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
-  var timeHour = document.createElement('td');
-  timeHour.textContent = firstAndPike.businessHours[cell];
-  timeRowEl.appendChild(timeHour);
+function creatingHeaderRow () {
```

Comment:
this looks to be duplicate code, you have another function called creatingHeaderRow on line 73